packages/tools/devtools/devtools-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -20,11 +20,18 @@ import {
 	ArrowJoinRegular,
 	ArrowExitRegular,
 } from "@fluentui/react-icons";
-
+import { ThemeContext } from "../ThemeHelper";
 import { clientIdTooltipText } from "./TooltipTexts";
 import { TransformedAudienceHistoryData } from "./AudienceView";
 import { LabelCellLayout } from "./utility-components";
 
+/**
+ * Returns the text color based on the current color theme of the devtools.
+ */
+function setThemeStyle(themeName: string): string {
+	return themeName === "highContrast" ? "#FFF" : "";
```

Comment:
Related to https://github.com/microsoft/FluidFramework/pull/16214, if we start using the enum for the stored value of the theme name instead of hardcoded strings, we'll want to update this to use the enum too.